index.bs

```diff
@@ -318,7 +233,190 @@ For example,
 is more readable than
 <code highlight="js">window.scrollBy(50, 0)</code>.
 
-<h3 id="promises">Design asynchronous APIs using Promises</h3>
+<h2 id="api-choices">Designing idiomatic and useful APIs</h2>
+
+<h3 id="constructors">Classes should have constructors when possible</h3>
+
+By default, [[!WEBIDL]] interfaces are reified in JavaScript as "non-constructible" classes: trying
```

Comment:
nit: "reified" seems like fancy jargon (specially because it conjures up "semantic web reification" 🤢). Is there a simpler term we can use here? If no, that's ok.

Reply:
represented?